rust/operator-binary/src/superset_controller.rs

```diff
@@ -733,39 +733,35 @@ fn build_server_rolegroup_statefulset(
             "-euo".to_string(),
             "pipefail".to_string(),
             "-c".to_string(),
-            formatdoc! {"
-                mkdir --parents {PYTHONPATH} && \
-                cp {STACKABLE_CONFIG_DIR}/* {PYTHONPATH} && \
-                cp {STACKABLE_LOG_CONFIG_DIR}/{LOG_CONFIG_FILE} {PYTHONPATH} && \
-                {auth_commands}
-                superset db upgrade && \
-                superset fab create-admin \
-                    --username \"$ADMIN_USERNAME\" \
-                    --firstname \"$ADMIN_FIRSTNAME\" \
-                    --lastname \"$ADMIN_LASTNAME\" \
-                    --email \"$ADMIN_EMAIL\" \
-                    --password \"$ADMIN_PASSWORD\" && \
-                superset init && \
-                {COMMON_BASH_TRAP_FUNCTIONS}
-                {remove_vector_shutdown_file_command}
-                prepare_signal_handlers
-                gunicorn \
-                --bind 0.0.0.0:${{SUPERSET_PORT}} \
-                --worker-class gthread \
-                --threads 20 \
-                --timeout {webserver_timeout} \
-                --limit-request-line 0 \
-                --limit-request-field_size 0 \
-                'superset.app:create_app()' &
-                wait_for_termination $!
-                {create_vector_shutdown_file_command}",
+        ])
+        .args(vec![formatdoc! {"
+            mkdir --parents {PYTHONPATH}
+            cp {STACKABLE_CONFIG_DIR}/* {PYTHONPATH}
+            cp {STACKABLE_LOG_CONFIG_DIR}/{LOG_CONFIG_FILE} {PYTHONPATH}
+
+            {auth_commands}
+
+            superset db upgrade
+            # Don't print the following statement, as it contains credentials
+            set +x
+            superset fab create-admin --username \"$ADMIN_USERNAME\" --firstname \"$ADMIN_FIRSTNAME\" --lastname \"$ADMIN_LASTNAME\" --email \"$ADMIN_EMAIL\" --password \"$ADMIN_PASSWORD\"
+            set -x
```

Comment:
Maybe lets still have an echo for e.g. `superset fab create-admin` or the full command with blanked password? No show stopper though, LGTM.

Reply:
Sure! Pls have a look at 39bb57c